binding/Binding/SKDrawable.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.IO;
+
+namespace SkiaSharp
+{
+	public abstract class SKDrawable : SKAbstractDrawable
+	{
+		[Preserve]
+		internal SKDrawable (IntPtr handle, bool owns)
+			: base (handle, owns)
+		{
+		}
+
+		public SKDrawable() : base ()
+		{
+		}
+
+		protected override void Dispose (bool disposing)
+		{
+			if (Handle != IntPtr.Zero && OwnsHandle) {
+				SkiaApi.sk_manageddrawable_destroy (Handle);
+			}
+
+			base.Dispose (disposing);
+		}
+
+		public uint GenerationID => SkiaApi.sk_manageddrawable_get_generation_id (Handle);
+
+		public SKRect Bounds {
+			get {
+				return SkiaApi.sk_manageddrawable_get_bounds (Handle);
+			}
+		}
+
+		public void Draw (SKCanvas canvas, ref SKMatrix matrix)
+		{
+			SkiaApi.sk_manageddrawable_draw (Handle, canvas.Handle, ref matrix);
+		}
+
+		public void Draw (SKCanvas canvas, float x, float y)
+		{
+			var matrix = SKMatrix.MakeTranslation (x, y);
+			SkiaApi.sk_manageddrawable_draw (Handle, canvas.Handle, ref matrix);
+		}
+
+		public SKPicture NewPictureSnapshot ()
```

Comment:
I am not sure what the best naming would be:

 - `ToPicture` (this is a nice one)
 - `AsPicture` (may imply that there is a simple cast, and not a totally disconnect)
 - `CreatePicture` (may be confusing with the past use of static constructor types)
 - `Snapshot` (used by `SKSurface` to indicate capturing the current state in an immutable form, so we have consistency here)

Any other ideas?

Reply:
I think we should go with `ToPicture` here, unless you have any other ideas.

```suggestion
		public SKPicture ToPicture ()
```